package.json

```diff
@@ -39,15 +44,19 @@
     "babel-preset-env": "^1.6.1",
     "clean-webpack-plugin": "^1.0.1",
     "copy-webpack-plugin": "^4.6.0",
+    "current-processes": "^0.2.1",
```

Comment:
I believe this is in the office-addins-test-helpers package now, so it can be removed here.